superset-frontend/package.json

```diff
@@ -275,10 +275,12 @@
     "jest": "^26.6.3",
     "jest-environment-enzyme": "^7.1.2",
     "jest-enzyme": "^7.1.2",
+    "jest-websocket-mock": "^2.2.0",
     "jsdom": "^16.4.0",
     "less": "^3.12.2",
     "less-loader": "^5.0.0",
     "mini-css-extract-plugin": "^0.4.0",
+    "mock-socket": "^9.0.3",
```

Comment:
Is this used anywhere?

Reply:
Apparently [it's a peer dependency](https://www.npmjs.com/package/jest-websocket-mock#install) of `jest-websocket-mock`.